examples/components/MNIST/aggregatemodelweights/aggregatemodelweights.yaml

```diff
@@ -29,11 +37,12 @@ outputs:
 code: .
 
 command: >-
-  python run.py --aggregated_output ${{outputs.aggregated_output}}
-  --input_silo_1 ${{inputs.input_silo_1}}
-  $[[--input_silo_2 ${{inputs.input_silo_2}}]]
-  $[[--input_silo_3 ${{inputs.input_silo_3}}]]
+  python run.py --output ${{outputs.aggregated_output}} 
+  --extension pt 
+  --checkpoints ${{inputs.input_silo_1}} 
+  $[[${{inputs.input_silo_2}}]] 
+  $[[${{inputs.input_silo_3}}]] 
+  $[[${{inputs.input_silo_4}}]] 
+  $[[${{inputs.input_silo_5}}]] 
 
-environment: 
-  conda_file: ./conda.yaml
-  image: mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04
+environment: azureml:AzureML-pytorch-1.10-ubuntu18.04-py38-cuda11-gpu:33
```

Comment:
It does not, the container can run on a gpu but it has all the dependencies to be able to run on a gpu.

Although, we should file a feature request so that the bicep provisioning provisions GPU as well.

Reply:
Filed as an issue here #169

Reply:
In this particular case, let me roll back the changes to use this default image instead, because we don't need the full gpu image at all.